ansible/lab

```diff
@@ -1,14 +1,21 @@
-[sonic_latest]
-str-msn2700-01    ansible_host=10.251.0.188 sonic_version=v2
+[sonic_mlnx_40]
+str-msn2700-01    ansible_host=10.251.0.188
+
+[sonic_mlnx_40:vars]
+hwsku="ACS-MSN2700"
+iface_prefix="Ethernet"
+iface_postfix=""
```

Comment:
remove iface_prefix and postfix, use port_config.ini in the buildimge repo to convert.